fluentd/version.txt

```diff
@@ -1 +1 @@
-1.13.3
+1.13.4
```

Comment:
This version should match the semver of fluentd, with a dash and number to indicate rebuilds of the same fluentd version. EG: in this case it would be `1.13.3-1`. However, you'll need to update this to whatever is generated by the `make` command suggested above.

Reply:
It was `Fetching fluentd 1.13.3` so, will add -1